tests/unit/matrix/test_measure.py

```diff
@@ -1064,7 +1087,7 @@ def it_computes_its_blocks_to_help(self, request, dimensions_):
             "_unweighted_cube_counts",
             return_value=unweighted_cube_counts_,
         )
-        SumSubtotals_ = class_mock(request, "cr.cube.matrix.measure.SumSubtotals")
+        SumSubtotals_ = class_mock(request, "cr.cube.matrix.measure.SumDiffSubtotals")
```

Comment:
This mock's name should also change, to `SumDiffSubtotals_` in this case. The idea is you should be able to tell everything important about the mock just from it's name, in this case, which class it is a mock _of_.